dynamiqs/integrators/core/diffrax_integrator.py

```diff
@@ -265,3 +266,46 @@ class MESolveDiffraxIntegrator(MEDiffraxIntegrator, SolveSaveMixin, SolveInterfa
 mesolve_kvaerno5_integrator_constructor = partial(
     MESolveDiffraxIntegrator, diffrax_solver=dx.Kvaerno5(), fixed_step=False
 )
+
+
+class MELindbladianDiffraxIntegrator(DiffraxIntegrator, MEInterface):
+    """Integrator computing the propagator the Lindblad master equation with Diffrax."""
+
+    @property
+    def terms(self) -> dx.AbstractTerm:
+        # define vector field for Lindblad eqn in Liousville super-operator form
+        # d/dt rho(t) = \mathcal{L}(\rho(t))
+
+        def vector_field(t, y, _):  # noqa: ANN001, ANN202
+            L, H = self.L(t), self.H(t)
+            return slindbladian(H, L) @ y
+
+        return dx.ODETerm(vector_field)
+
+
+class MEPropagatorDiffraxIntegrator(
+    MELindbladianDiffraxIntegrator, PropagatorSaveMixin
+):
+    """Integrator computing the propagator of the Lindblad master equation
+    using the Diffrax library.
+    """
```

Comment:
Since `MELindbladianDiffraxIntegrator` is only used in `MEPropagatorDiffraxIntegrator` (at least for now) it might make sense to merge the classes:
```suggestion
class MEPropagatorDiffraxIntegrator(DiffraxIntegrator, MEInterface, PropagatorSaveMixin):
    """Integrator computing the propagator of the Lindblad master equation with Diffrax."""

    @property
    def terms(self) -> dx.AbstractTerm:
        # define vector field for Lindblad eqn in Liousville super-operator form
        # d/dt rho(t) = \mathcal{L}(\rho(t))

        def vector_field(t, y, _):  # noqa: ANN001, ANN202
            L, H = self.L(t), self.H(t)
            return slindbladian(H, L) @ y

        return dx.ODETerm(vector_field)
```

Reply:
OK sounds good - thank you.